### PR TITLE
Update configuring-notifications.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Reviewed content based on https://github.com/mattermost/docs/issues/3187, and made some stylistic changes.

### CODE REVIEW

File: source/help/getting-started/configuring-notifications.rst [hunk 1]
```diff
@@ -63,14 +52,15 @@ Mobile push notifications appear on the lock screen of your mobile device if the
 Browser Tab Notifications
 ----------------------------------------
 
-If Mattermost is open in a browser tab, the favicon updates to notify you of unread messages (\*) and
-a count of mentions or Direct Messages. Browser tab notifications are available on Firefox and Chrome.
+If Mattermost is open in a browser tab, the favicon updates to notify you of unread messages (\*) and a count of mentions or Direct Messages. Browser tab notifications are available on Firefox and Chrome.
```
Comment (justinegeffen) [inline source/help/getting-started/configuring-notifications.rst]: Awaiting confirmation as to whether this should include Edge and Safari.

**Comment by jasonblais** [comment]
Thanks @justinegeffen! Feel free to request PM review once it's ready for review. Best PM for notifications will be Eric Sethna.

File: source/help/getting-started/configuring-notifications.rst [hunk 2]
```diff
@@ -63,14 +52,15 @@ Mobile push notifications appear on the lock screen of your mobile device if the
...
 
 .. image:: ../../images/browser_notification.png
 
 Muting a Channel
 ----------------------------------------
 
-Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear unread and you do not recieve notifications, click the channel name at the top of the page to access the **Channel Menu** > **Notification Preferences > Mute channel**.
+Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear as unread unless you are mentioned, click the channel name at the top of the page and navigate to **Notification Preferences > Mute channel**, where you can edit the notification settings for that channel.
```
Comment (esethna) [inline source/help/getting-started/configuring-notifications.rst]: Not sure this answers the orignal issue brought up by the github issue, responded there, feel free to make any other changes necessary

https://github.com/mattermost/docs/issues/3187#issuecomment-562241056

File: source/help/getting-started/configuring-notifications.rst (hunk 1 above)
Comment (justinegeffen) [reply source/help/getting-started/configuring-notifications.rst]: Sure thing. :)

Comment (justinegeffen) [inline source/help/getting-started/configuring-notifications.rst:55]:
```suggestion
If Mattermost is open in a browser tab, the favicon updates to notify you of unread messages (\*) and a count of mentions or Direct Messages. Browser tab notifications are available on Chrome, Edge, Firefox, Safari.
```

Comment (justinegeffen) [reply source/help/getting-started/configuring-notifications.rst]: Confirmed that behavior is supported on Edge and Safari; content updated.

File: source/help/getting-started/configuring-notifications.rst [hunk 3]
```diff
@@ -1,32 +1,27 @@
 Configuring Notifications
 =========================
 
-Desktop, email and mobile push notifications notify you of activity in Mattermost. When you join a team, Mattermost will notify you of messages directed at you, including when someone:
+Desktop, email, and mobile push notifications notify you of activity in Mattermost. When you join a team, Mattermost will notify you of messages directed at you, including when someone:
 
-- Direct Messages you
+- Sends you a Direct Messages
```
Comment (amyblais) [inline source/help/getting-started/configuring-notifications.rst:6]:
```suggestion
- Sends you a Direct Message
```

File: source/help/getting-started/configuring-notifications.rst [hunk 4]
```diff
@@ -63,14 +52,28 @@ Mobile push notifications appear on the lock screen of your mobile device if the
...
 
 .. image:: ../../images/browser_notification.png
 
 Muting a Channel
 ----------------------------------------
 
-Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear unread and you do not recieve notifications, click the channel name at the top of the page to access the **Channel Menu** > **Notification Preferences > Mute channel**.
+Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. 
+
+When you mute a channel, you do not receive any notifications (desktop, email, or push) for any mentions (whether they are channel-wide to directed at you). However, if you are mentioned while the channel is muted the mention badge will be displayed in the sidebar with the channel displaying at reduced opacity.
```
Comment (amyblais) [inline source/help/getting-started/configuring-notifications.rst:64]:
```suggestion
When you mute a channel, you do not receive any notifications (desktop, email, or push) for any mentions (whether they are channel-wide or directed at you). However, if you are mentioned while the channel is muted, the mention badge will be displayed in the sidebar with the channel displaying at reduced opacity.
```

File: source/help/getting-started/configuring-notifications.rst [hunk 5]
```diff
@@ -63,14 +52,28 @@ Mobile push notifications appear on the lock screen of your mobile device if the
...
+When you mute a channel, you do not receive any notifications (desktop, email, or push) for any mentions (whether they are channel-wide to directed at you). However, if you are mentioned while the channel is muted the mention badge will be displayed in the sidebar with the channel displaying at reduced opacity.  
+
+To mute a channel, click the channel name at the top of the page and navigate to **Notification Preferences > Mute channel**, where you can edit the notification settings for that channel. You can also click on the channel name and select **Mute Channel**. To unmute it, click on the channel name and select **Unmute Channel**.
 
 .. tip :: Mentions are triggered by `selected keywords <https://docs.mattermost.com/help/settings/account-settings.html#words-that-trigger-mentions>`_. Learn more about `mentioning teammates <http://docs.mattermost.com/help/messaging/mentioning-teammates.html>`__.
+
+Ignoring Mentions
+-----------------
+
+To turn off notifications for channel-wide mentions for @channel, @here, and @all navigate to **Notification Preferences > Ignore mentions for @channel, @here and @all**. Choose **Edit**, select **On**, and then select **Save**.
```
Comment (amyblais) [inline source/help/getting-started/configuring-notifications.rst:73]:
```suggestion
To turn off notifications for channel-wide mentions for @channel, @here, and @all, navigate to **Notification Preferences > Ignore mentions for @channel, @here and @all**. Choose **Edit**, select **On**, and then select **Save**. 
```

File: source/help/getting-started/configuring-notifications.rst (hunk 2 above)
Comment (esethna) [reply source/help/getting-started/configuring-notifications.rst]: @justinegeffen see comment above ^. Also you don't need to go to the notification preferences anymoe, mute is in the dropdown menu directly

File: source/help/getting-started/configuring-notifications.rst [hunk 6]
```diff
@@ -63,14 +52,28 @@ Mobile push notifications appear on the lock screen of your mobile device if the
...
 ----------------------------------------
 
-Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear unread and you do not recieve notifications, click the channel name at the top of the page to access the **Channel Menu** > **Notification Preferences > Mute channel**.
+Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. 
+
+When you mute a channel, you do not receive any notifications (desktop, email, or push) for any mentions (whether they are channel-wide to directed at you). However, if you are mentioned while the channel is muted the mention badge will be displayed in the sidebar with the channel displaying at reduced opacity.  
+
+To mute a channel, click the channel name at the top of the page and navigate to **Notification Preferences > Mute channel**, where you can edit the notification settings for that channel. You can also click on the channel name and select **Mute Channel**. To unmute it, click on the channel name and select **Unmute Channel**.
```
Comment (esethna) [inline source/help/getting-started/configuring-notifications.rst:66]: Do we need to state both methods here or just the easiest one?

File: source/help/getting-started/configuring-notifications.rst [hunk 7]
```diff
@@ -63,14 +52,28 @@ Mobile push notifications appear on the lock screen of your mobile device if the
...
 ----------------------------------------
 
-Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear unread and you do not recieve notifications, click the channel name at the top of the page to access the **Channel Menu** > **Notification Preferences > Mute channel**.
+Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. 
+
+When you mute a channel, you do not receive any notifications (desktop, email, or push) for any mentions (whether they are channel-wide or directed at you). However, if you are mentioned while the channel is muted, the mention badge will be displayed in the sidebar with the channel displaying at reduced opacity.
+
+To mute a channel, click the channel name at the top of the page and navigate to **Notification Preferences > Mute channel**, where you can edit the notification settings for that channel. You can also click on the channel name and select **Mute Channel**. To unmute it, click on the channel name and select **Unmute Channel**.
```
Comment (justinegeffen) [inline source/help/getting-started/configuring-notifications.rst:66]:
```suggestion
To mute a channel, click on the channel name and select **Mute Channel**. To unmute it, click on the channel name and select **Unmute Channel**.
```

File: source/help/getting-started/configuring-notifications.rst (hunk 6 above)
Comment (justinegeffen) [reply source/help/getting-started/configuring-notifications.rst:66]: Good catch, I've edited it to use the easiest one.

File: source/help/getting-started/configuring-notifications.rst (hunk 2 above)
Comment (justinegeffen) [reply source/help/getting-started/configuring-notifications.rst]: Re-reading this comment, I think it has been addressed by the latest changes. However, if not, let me know and I can rework the update.

Comment (justinegeffen) [reply source/help/getting-started/configuring-notifications.rst]: Following up on this to confirm whether the comment has been addressed or reviewed.

**Pull request merged**